nvflare/fuel/hci/server/hci.py

```diff
@@ -107,7 +107,7 @@ def __init__(
             server_key: server's private key file
             accepted_client_cns: list of accepted Common Names from client, if specified
         """
-        socketserver.TCPServer.__init__(self, (host, port), _MsgHandler, False)
+        socketserver.TCPServer.__init__(self, ("0.0.0.0", port), _MsgHandler, False)
```

Comment:
why do we change this from "host" to "0.0.0.0"?

Reply:
In k8s, the host name is controlled by the cluster.  Binding 0.0.0.0 is a working solution so the server can listen to the port.

Reply:
should we get rid of this "host" arg is it is not used?
@yhwen you see any issues removing this host arg?

Reply:
I think we should keep that arg right now.  The information could be used somewhere else.